details/fls-engine.md

```diff
@@ -1,6 +1,6 @@
 # FLS engine
 
-Applicable in the context of the [`fields` rule](elasticsearch.md#fields) 
```

Comment:
is this intentional?

Reply:
Yes, `elasticsearch.md` will try to open github page with result 404 not found 

![image](https://github.com/user-attachments/assets/9a9be156-3294-42d4-8572-69f400d0b019)